{"level":"ERROR","location":"[unsubscribe:1756] data_ws","message":"11","timestamp":"2025-03-05 12:25:34,661+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Users\\JAIMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1733, in unsubscribe\n    if symb not in self.scrips_count[self.channel_num]:\n                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\nKeyError: 11","exception_name":"KeyError"}
{"level":"ERROR","location":"[unsubscribe:1756] data_ws","message":"11","timestamp":"2025-03-05 12:27:53,654+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Users\\JAIMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1733, in unsubscribe\n    if symb not in self.scrips_count[self.channel_num]:\n                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\nKeyError: 11","exception_name":"KeyError"}
{"level":"ERROR","location":"[unsubscribe:1756] data_ws","message":"11","timestamp":"2025-03-05 12:29:15,955+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Users\\JAIMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1733, in unsubscribe\n    if symb not in self.scrips_count[self.channel_num]:\n                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\nKeyError: 11","exception_name":"KeyError"}
{"level":"ERROR","location":"[unsubscribe:1756] data_ws","message":"11","timestamp":"2025-03-05 12:30:21,982+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Users\\JAIMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1733, in unsubscribe\n    if symb not in self.scrips_count[self.channel_num]:\n                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\nKeyError: 11","exception_name":"KeyError"}
{"level":"ERROR","location":"[unsubscribe:1756] data_ws","message":"11","timestamp":"2025-03-05 12:32:31,295+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Users\\JAIMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1733, in unsubscribe\n    if symb not in self.scrips_count[self.channel_num]:\n                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\nKeyError: 11","exception_name":"KeyError"}
{"level":"ERROR","location":"[unsubscribe:1756] data_ws","message":"11","timestamp":"2025-03-05 12:34:45,296+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Users\\JAIMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1733, in unsubscribe\n    if symb not in self.scrips_count[self.channel_num]:\n                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\nKeyError: 11","exception_name":"KeyError"}
{"level":"ERROR","location":"[unsubscribe:1756] data_ws","message":"11","timestamp":"2025-03-05 12:36:36,289+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Users\\JAIMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1733, in unsubscribe\n    if symb not in self.scrips_count[self.channel_num]:\n                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\nKeyError: 11","exception_name":"KeyError"}
{"level":"ERROR","location":"[unsubscribe:1756] data_ws","message":"11","timestamp":"2025-03-13 10:49:04,527+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Users\\JAIMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1733, in unsubscribe\n    if symb not in self.scrips_count[self.channel_num]:\n                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\nKeyError: 11","exception_name":"KeyError"}
